saliency/base.py

```diff
@@ -95,3 +98,83 @@ def GetMask(self, x_value, feed_dict={}):
     """
     feed_dict[self.x] = [x_value]
     return self.session.run(self.gradients_node, feed_dict=feed_dict)[0]
+
+
+OUTPUT_GRADIENTS = 'OUTPUT_GRADIENTS'
+CONVOLUTION_LAYER = 'CONVOLUTION_LAYER'
+CONVOLUTION_GRADIENTS = 'CONVOLUTION_GRADIENTUS'
+
+
+class CallModelSaliency(object):
+  r"""Base class for saliency masks. Alone, this class doesn't do anything."""
+
+  def __init__(self):
```

Comment:
It is better not to define \_\_init\_\_ method if it doesn't do anything.

Reply:
Removed.